react-native/react/tracker/proofs.render.desktop.js

```diff
@@ -32,25 +33,47 @@ export default class ProofsRender extends Component {
   }
 
   renderPlatformProof (pp) {
-    const name = pp.platform.name === 'web' ? pp.platform.uri : pp.platform.name
-    console.log(name)
+    const name = pp.platform.name === 'web' ? pp.platform.uri : pp.platform.username
```

Comment:
This was changed in my stuff. Remember we didn't want to do this kind of translation logic here